README.md

```diff
@@ -175,6 +176,7 @@ cmake --build . --target all
 | `FLAMEGPU_RTC_DISK_CACHE`            | `ON`/`OFF`                  | Enable/Disable caching of RTC functions to disk. Default `ON`.                                             |
 | `FLAMEGPU_VERBOSE_PTXAS`             | `ON`/`OFF`                  | Enable verbose PTXAS output during compilation. Default `OFF`.                                             |
 | `FLAMEGPU_CURAND_ENGINE`             | `XORWOW` / `PHILOX` / `MRG` | Select the CUDA random engine. Default `XORWOW`                                                            |
+| `FLAMEGPU_ENABLE_MPI`                | `ON`/`OFF`                  | Enable MPI support for distributed CUDAEnsembles. Default `OFF`.                                           |
```

Comment:
The restriction on <= one mpi rank per GPU per node (?) needs to be very clearly spelled out in the readme (and the docs).

Reply:
It's recommended 1 MPI rank per node, FGPU can handle dividing up multiple GPUs (and this reduces comms).

Don't recall how multi gpu/rank is handled, should be noted in docs but will address here too.

Reply:
Have said MPI worker needs exclusive access to gpus, each 1 per node.